Users/Tutorial.rst

```diff
@@ -170,11 +170,14 @@ Thats all nice and well but we also have a Makefile for our project we
 want to check. So let us introduce another feature of our configuration
 syntax: *sections*.
 
-The line ``[Default]`` specifies that everything below will belong to
-the Default section. If nothing is specified, a setting will implicitly
-belong to this section.
+The line ``[cli]`` implies that everything below belongs to the "cli"
+section. This section handles all the settings that are entered via the
+Command Line Interface (CLI) and that don't specify any particular
+section in them (you have not specified any particular sections till
```

Comment:
"if" missing? --> (**if** yyou have not...

Reply:
Actually, what I am trying to tell that if the reader has been following the tutorial till now, then he/she has not specified any particular section (because we have not presented the `sections` feature till this point in the tutorial). Maybe its not so clear :( I'll rephrase the sentence.